wallets/shared/src/rango.ts

```diff
@@ -157,13 +157,73 @@ export enum Namespace {
   Tron = 'Tron',
 }
 
-export const namespaceMainBlockchain: Record<Namespace, string> = {
-  [Namespace.Evm]: 'ETH',
-  [Namespace.Solana]: 'SOLANA',
-  [Namespace.Cosmos]: 'COSMOS',
-  [Namespace.Utxo]: 'BTC',
-  [Namespace.Starknet]: 'STARKNET',
-  [Namespace.Tron]: 'TRON',
+export type DerivationPath = {
+  id: string;
+  label: string;
+  generateDerivationPath: (index: string) => string;
+};
+
+export const namespaces: Record<
```

Comment:
Why you defined this here? this value is using for namespace and derivation path modals. if this is true, you could put this value into `embededed` itself.

Reply:
You are right. Moved it to `widget/embedded/src/constants/namespaces/`